--- old/.venv/lib/python3.12/site-packages/mmagic/__init__.py	2024-09-22 22:13:12.871210390 +0200
+++ new/.venv/lib/python3.12/site-packages/mmagic/__init__.py	2024-09-22 22:19:51.818553644 +0200
@@ -21,7 +21,7 @@
 
 
 MMCV_MIN = '2.0.0'
-MMCV_MAX = '2.2.0'
+MMCV_MAX = '2.3.0'
 mmcv_min_version = digit_version(MMCV_MIN)
 mmcv_max_version = digit_version(MMCV_MAX)
 mmcv_version = digit_version(mmcv.__version__)
