--- ./.venv/lib/python3.12/site-packages/ultralytics/models/yolo/segment/predict.py	2025-01-05 17:33:19.194644332 +0100
+++ new/.venv/lib/python3.12/site-packages/ultralytics/models/yolo/segment/predict.py	2025-01-05 17:37:32.288827204 +0100
@@ -35,6 +35,7 @@
             max_det=self.args.max_det,
             nc=len(self.model.names),
             classes=self.args.classes,
+            max_time_img=0.3
         )
 
         if not isinstance(orig_imgs, list):  # input images are a torch.Tensor, not a list
